app/assets/javascripts/admin/addon/templates/config-look-and-feel.hbs

```diff
@@ -1,9 +1,10 @@
-<AdminPageHeader
+<DPageHeader
   @titleLabel="admin.config_areas.look_and_feel.title"
   @descriptionLabel="admin.config_areas.look_and_feel.description"
   @learnMoreUrl="https://meta.discourse.org/t/beginners-guide-to-using-discourse-themes/91966"
 >
   <:breadcrumbs>
+    <DBreadcrumbsItem @path="/admin" @label={{i18n "admin_title"}} />
```

Comment:
that's a detail but in ember it's generally called route not path

Reply:
To me `@route` refers to an actual ember route whereas `@path` is just the end part of the URL. We did use a route initially for these, but I worked with  CvX on it and it was causing too many errors on page transitions so we went with a raw string route instead.

Would prefer to keep this as `@path` for now unless you feel strongly